Cogs.Console/Program.cs

```diff
@@ -163,7 +163,6 @@ static void Main(string[] args)
                
                 var locationArgument = command.Argument("[cogsLocation]", "Directory where the COGS datamodel is located.");
                 var targetArgument = command.Argument("[targetLocation]", "Directory where the dot schema is generated.");
-                var dotArgument = command.Argument("[dotLocation]", "Directory where the dot.exe file is located.");
```

Comment:
We want to be able to specify the location of dot., not embed it.
Cogs runs on windows, mac, and linux and dot it will be different per platform.
For the tests, we can assume that it is in the environmental Path on the machine, and add it to our development machines